systemd/usr/share/xdg-desktop-portal/portals/wlr.portal

```diff
@@ -0,0 +1,4 @@
+[portal]
+DBusName=org.freedesktop.impl.portal.desktop.wlr
+Interfaces=org.freedesktop.impl.portal.Screenshot;org.freedesktop.impl.portal.ScreenCast;
+UseIn=wlroots;sway;Wayfire;river;phosh;Hyprland;Miriway
```

Comment:
```suggestion
UseIn=Mir
```

Reply:
And then make `XDG_CURRENT_DESKTOP=Miriway:Mir`, this way snap downstreams can benefit from this straight up?

Reply:
Miracle should set `XDG_CURRENT_DESKTOP=Miracle:Mir`, no? It's not matched by `startswith`, AIUI?

Or do you mean it already has `:mir`?

Reply:
I suppose we could just rename it `mir.portal` and install outside of snaps, too